docs/drafts/support-watchNamespaces.md

```diff
@@ -0,0 +1,24 @@
+# Install Modes and WatchNamespaces in OMLv1
+
+Operator Lifecycle Manager (OLM) operates with cluster-admin privileges, enabling it to grant necessary permissions to the Operators it deploys. For operators packaged as [`RegistryV1`][registryv1] bundles, it's the responsibility of the authors to specify supported `InstallModes` in the ClusterServiceVersion ([CSV][csv]). InstallModes define the operational scope of the Operator within the Kubernetes cluster, particularly in terms of namespace availability. The four recognized InstallModes are as follows:
+
+1. OwnNamespace: This mode allows the operator to monitor and respond to events within its own deployment namespace.
+1. SingleNamespace: In this mode, the operator is set up to observe events in a single, specific namespace other than the one it is deployed in.
+1. MultiNamespace: This enables the operator to function across multiple specified namespaces.
+1. AllNamespaces: Under this mode, the operator is equipped to monitor events across all namespaces within the cluster.
+
+When creating a cluster extension for deploying an operator, users have the option to define a list of `watchNamespaces`. This list determines the specific namespaces within which they intend the operator to operate. The configuration of `watchNamespaces` must align with the InstallModes supported by the operator as specified by the bundle author. The supported configurations in the order of preference are as follows:
+
```

Comment:
Using the word "Operators" for now. Not sure of the naming convention we want to follow. ("Extension" vs "Operators")

Reply:
My approach for the rename was:
- If we're talking about this API or a bundle in general, those are extensions.
- If we're talking specifically about the "Operator" pattern, which is a specific type of extension, it is okay to use "operator" terminology.

Since we're adding a "watchNamespaces" field directly to the ClusterExtension API, I think we're saying "extensions are things that may be able to be configured to watch specific namespaces", so I don't think that watching namespaces is strictly an operator concept.

Reply:
Modified to be "Extension" now.